packages/devextreme/js/__internal/ui/chat/chat_message_box.ts

```diff
@@ -78,6 +84,17 @@ class MessageBox extends DOMComponent<MessageBox, Properties> {
       placeholder: 'Type a message',
       autoResizeEnabled: true,
       maxHeight: '20em',
+      onEnterKey: (e: EnterKeyEvent): void => {
+        if (!e.event?.shiftKey) {
+          this._sendHandler(e);
+        }
+      },
+    });
+
+    this._textArea.registerKeyHandler('enter', (event: KeyboardEvent) => {
+      if (!event.shiftKey && this._isValuableTextEntered()) {
+        event.preventDefault();
```

Comment:
Please, check the following scenario: 
- press enter
- type text
- press enter

TextArea don't return to initial height, but it's empty

<img width="478" alt="image" src="https://github.com/user-attachments/assets/2925c8f5-ba74-45c8-b80b-682a0f153d65">

Reply:
yeah, after textarea get `autoResizeEnabled` it stopped working. Seems like we need setting `valueChangeEvent` to `input` now

Reply:
[commit](https://github.com/DevExpress/DevExtreme/pull/28003/commits/8d70833dd1bd99f2e1e5ac5b6f4ff5102c42bb71)